### PR TITLE
Feature/condo/doma 2269/feature select filter component 2

### PR DESCRIPTION
**Pull request by VKislov**

### CODE REVIEW

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/core'
+import { colors } from '../../constants/style'
+
+
+export const useApplySelectStyledCss = (applyCss: boolean, raisedClearButton: boolean) => {
+
+    const greyStyle = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts:7]: it's constant actually and can be moved outside the function

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        background-color: ${colors.ultraLightGrey};
+        border-radius: 8px
+    }
+
+    &.ant-select-lg.ant-select-multiple .ant-select-selection-item {
+        background-color: ${colors.white};
+    }
+    `
+
+    const borderedClrBtnStyle = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts:20]: same

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    .ant-select-clear {
+        border-radius: 5px
+    }
+    `
+
+    const raisedClrBtnStyle = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts:26]: same

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    .ant-select-clear {
+        top: 30%;
+    }
+    `
+    const style: any = []
+    
+    if (applyCss){
```
Comment (akarjakin) [inline apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts:33]: this naming is not obvious, you can't simply understand what will happen only by seeing prop name, you'll have to read whole code.

File: apps/condo/domains/common/components/SelectStyled/index.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    const { 
+        applyCss,
+        raisedClearButton, 
+        ...otherSelectProps 
+    } = props
+
+    const  style = useApplySelectStyledCss(applyCss, raisedClearButton)
+
+    return (
+        <>
```
Comment (akarjakin) [inline apps/condo/domains/common/components/SelectStyled/index.tsx:22]: you don't need fragment here

File: apps/condo/domains/common/components/Pickers/DateRangePicker.tsx [hunk 6]
```diff
@@ -6,12 +6,18 @@ import dayjs, { Dayjs } from 'dayjs'
 import { DownOutlined, MinusOutlined } from '@ant-design/icons'
 import { fontSizes } from '@condo/domains/common/constants/style'
 import { RangePickerSharedProps } from 'rc-picker/lib/RangePicker'
+import { colors } from '../../constants/style'
+
 
 const rangePickerCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/Pickers/DateRangePicker.tsx:12]: this is constant, so `RANGE_PICKER_CSS`

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 7]
```diff
@@ -10,73 +10,74 @@ export const COLLAPSED_SIDE_MENU_WIDTH = 92
 
 // Ant inputs
 export const formInputFixCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:12]: it's constant

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 8]
```diff
@@ -10,73 +10,74 @@ export const COLLAPSED_SIDE_MENU_WIDTH = 92
...
+      border-color: ${red[5]};
+      background-color: ${colors.white};
+    }
+    .ant-input:focus:hover, .ant-input:active:hover {
+      background-color: ${colors.white};
+    }
 `
 
+
 export const mobileSideNavStyles = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:47]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 9]
```diff
@@ -10,73 +10,74 @@ export const COLLAPSED_SIDE_MENU_WIDTH = 92
...
-  z-index: ${zIndex.mobileSidenav};
+    position: fixed;
+    padding: 16px 0 60px;
+    box-sizing: border-box;
+    overflow-y: scroll;
+    height: 100vh;
+    z-index: ${zIndex.mobileSidenav};
 `
 
 export const sideNavStyles = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:56]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 10]
```diff
@@ -86,54 +87,54 @@ export const MenuItemsContainer = styled.div`
...
+    }
+    &:hover {
+        background: rgba(0, 0, 0, 0.025);
+    }
+    .avatar {
+        margin-right: 8px;
+    }
 `
 
 export const menuIconStyles = {
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:140]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 11]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
-    background-color: ${colors.defaultWhite[5]};
-    font-weight: 700;
-  }
+    &:hover {
+        background-color: ${colors.defaultWhite[5]};
+        font-weight: 700;
+    }
 `
 
 export const layoutCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:169]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 12]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
-  height: 100%;
-  display: flex;
-  align-items: stretch;
+    height: 100%;
+    display: flex;
+    align-items: stretch;
 `
 
 export const subLayoutCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:175]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 13]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
-  display: flex;
-  align-items: stretch;
-  background-color: ${colors.white};
+    width: 100%;
+    display: flex;
+    align-items: stretch;
+    background-color: ${colors.white};
 `
 
 export const topMenuCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:182]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 14]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
-  display: flex;
-  flex-direction: column;
+    padding: ${({ isSmall }) => isSmall ? '20px 20px 0' : '20px 48px 0'};
+    margin: 0;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
 `
 
 export const pageHeaderCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:206]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 15]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
-  max-width: 1200px;
-  padding-bottom: 56px;
-  background: ${colors.white};
+    flex-grow: 1;
+    max-width: 1200px;
+    padding-bottom: 56px;
+    background: ${colors.white};
 `
 
 export const tablePageContentCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:223]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 16]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
 `
 
 export const spacedPageHeaderCss = css`
-  padding: 0 0 60px;
-  background: ${colors.white};
+    padding: 0 0 60px;
+    background: ${colors.white};
 `
 
 export const pageContentCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:216]: same

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 17]
```diff
@@ -142,86 +143,86 @@ export const menuIconStyles = {
...
 `
 
 export const pageHeaderCss = css`
-  padding: 0 0 40px;
-  background: ${colors.white};
+    padding: 0 0 40px;
+    background: ${colors.white};
 `
 
 export const spacedPageHeaderCss = css`
```
Comment (akarjakin) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:211]: same

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts (hunk 1 above)
Comment (VKislov) [reply apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts]: OK, thx. Corrected

File: apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts (hunk 2 above)
Comment (VKislov) [reply apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts]: OK, look at the latest commits plz.

**Comment by sitozzz** [comment]
Could you please provide some screenshots/gif's 'before -> after' of what exactly did you fix?

**Comment by VKislov** [comment]
image
Before 

Screenshot from 2022-04-29 13-23-43
After.

**Pull request merged**